checker/src/org/checkerframework/checker/interning/qual/Interned.java

```diff
@@ -26,16 +28,18 @@
  * annotated.
  * <p>
  *
- * This annotation is associated with the {@link InterningChecker}.
+ * This annotation is associated with the {@link org.checkerframework.checker.interning.InterningChecker}.
```

Comment:
I don't think the fully-qualified names are necessary here (and I find them a bit distracting to read), if you retain the import statement.  But maybe this change is necessary in some version of Java?  Or maybe you made the change because your IDE incorrectly removes the import statement because it doesn't look in Javadoc comments, and you are using the fully-qualified name in the Javadoc comments to work around that problem?  I'm just wondering the rationale for the change.